From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: SoSeDiK <mrsosedik@gmail.com>
Date: Mon, 9 Sep 2024 21:15:50 +0300
Subject: [PATCH] Pufferfish's yaml changes

Include Simple-YAML repo and sync snakeyaml version to API's one

diff --git a/build.gradle.kts b/build.gradle.kts
index 82ca36909d4f4e4910f23812fbe2c22fdd7ab957..f66407a02d39785b5f6980f1299a4402c14c1b81 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -24,6 +24,12 @@ abstract class MockitoAgentProvider : CommandLineArgumentProvider {
 }
 // Paper end - configure mockito agent that is needed in newer java versions
 
+// Kiterino start - Pufferfish's yaml changes
+repositories {
+    maven("https://jitpack.io") // Pufferfish's Simple-YAML
+}
+// Kiterino end - Pufferfish's yaml changes
+
 dependencies {
     implementation(project(":kiterino-api")) // Kiterino - Init
     // Paper start
@@ -61,7 +67,7 @@ dependencies {
     runtimeOnly("org.apache.maven.resolver:maven-resolver-transport-http:1.9.18")
 
     // Pufferfish start
-    implementation("org.yaml:snakeyaml:1.32")
+    implementation("org.yaml:snakeyaml:2.2") // Kiterino - Sync snakeyaml to API's version - Pufferfish's yaml changes
     implementation ("com.github.carleslc.Simple-YAML:Simple-Yaml:1.8.4") { // Purpur
         exclude(group="org.yaml", module="snakeyaml")
     }
